src/packet.rs

```diff
@@ -235,7 +237,8 @@ impl<'a> RtpPacket<'a> {
         }
     }
 
-    fn payload_offset(&self) -> usize {
+    /// Returns the offset of the payload in this packet relative to the beginning of the packet.
+    pub fn payload_offset(&self) -> usize {
```

Comment:
Is there a particular reason you need this public?  Just trying to understand what this gives over accessing the payload/extension using the already provided API?

Reply:
I want to keep a range around for the payload so I can easily access it at a later time without the lifetime constraints of only having a `&[u8]`. Currently I work around this with some pointer arithmetic to calculate the range from the two slices: https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/merge_requests/1424/diffs#0f3a605f32b7f9282e35bac4651a296b7d692f50_0_1193